comms/email.md

```diff
@@ -0,0 +1,13 @@
+Hello everyone,
+
+We're looking forward to seeing you all at the What They Forgot to Teach You About R workshop at {LOCATION} on {DATES}. This email covers important setup that you'll need to do before the workshop.
+
+To fully participate in the workshop, you'll need to bring with you a laptop you can install R packages on and that has:
+
+1. A recent version of R (we recommend 3.6+)
+2. A recent version of RStudio (we recommend 1.2.5000+; versions earlier than 1.1 will _not_ work)
+3. Git and GitHub set up and connected to RStudio
```

Comment:
Maybe mention PAT? Because someone who has Git/GitHub/RStudio already set up might think they don't need to do any prep. Perhaps:

> Git and GitHub set up and connected to RStudio (including a GitHub personal access token)